src/modules/widgets/constants.ts

```diff
@@ -9,6 +9,8 @@ export const SET_WIDGET_INITIALIZATION = '@widgets/SET_WIDGET_INITIALIZATION';
 export const INITIALIZE_WIDGET = '@widgets/INITIALIZE_WIDGET';
 export const INITIALIZE_CHART_WIDGET = '@widgets/INITIALIZE_CHART_WIDGET';
 export const EDIT_CHART_WIDGET = '@widgets/EDIT_CHART_WIDGET';
+export const EDIT_IMAGE_WIDGET = '@widgets/EDIT_IMAGE_WIDGET';
+export const IMAGE_WIDGET_CONFIGURE = '@widgets/IMAGE_WIDGET_CONFIGURE';
```

Comment:
please follow the naming convention, so the verb (action) should go first `CONFIGURE_IMAGE_WIDGET`
might be `SET_IMAGE_WIDGET` as it's shorter ;)